### PR TITLE
[11.0][ADD]stock_orderpoint_procure_location

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
This module allows users to define the procure location to be used in procurements created from minimum stock rules.

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
cc @ForgeFlow

File: stock_orderpoint_procure_location/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    "summary": "Allows to create procurement orders in a different location"
+    "the orderpoint",
+    "version": "11.0.1.0.0",
+    "author": "ForgeFlow, " "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
+    "depends": ["purchase", "stock"],
+    "data": ["views/stock_warehouse_orderpoint_view.xml"],
+    "license": "AGPL-3",
+    "development_status": "Alpha",
```
Comment (LoisRForgeFlow) [inline stock_orderpoint_procure_location/__manifest__.py]: Change this to Beta at least

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM, one minor comment.

**Comment by LoisRForgeFlow** [comment]
@AaronHForgeFlow could you have a look at travis?

**Comment by AaronHForgeFlow** [comment]
@LoisRForgeFlow Travis fixed here: https://github.com/OCA/stock-logistics-warehouse/pull/1293

**Comment by LoisRForgeFlow** [comment]
Thanks!

/ocabot merge nobump

**Pull request merged**